### ISSUE TITLE
Unhandled excception when importing a PDF, may be linked to PDF/A format

### ISSUE THREAD

**Issue by jmmelko** [JabRef/jabref#9204]
### JabRef version

5.7 (latest release), or 5.8

### Operating system

Windows

### Details on version and operating system

JabRef 5.8--2022-10-02--1797af8
Windows 10 10.0 amd64 
Java 18.0.2.1 
JavaFX 19+11

### Checked with the latest development build

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

1. Download this pdf: https://www.dropbox.com/s/0vtyhir0303pbuy/oe-30-4-5916.pdf?dl=0
2. Click import and select the file
3. You should get: `Cannot invoke "String.trim()" because the return value of "org.apache.xmpbox.type.PDFASchemaType.getNamespaceURI()" is null`
4. Active modification in acrobat reader, and save the file as a normal pdf (not PDF/A Archive)
5. It works
6. Recreate a PDF/A back using Acrobat Pro
7. The problem disappears 

In any case I think that Jabref should handle null returns from org.apache.com

thanks

### Appendix

<details>

```
java.lang.NullPointerException: Cannot invoke "String.trim()" because the return value of "org.apache.xmpbox.type.PDFASchemaType.getNamespaceURI()" is null
	at org.jabref.merged.module@5.8.127/org.apache.xmpbox.xml.PdfaExtensionHelper.populatePDFASchemaType(Unknown Source)
	at org.jabref.merged.module@5.8.127/org.apache.xmpbox.xml.PdfaExtensionHelper.populateSchemaMapping(Unknown Source)
	at org.jabref.merged.module@5.8.127/org.apache.xmpbox.xml.DomXmpParser.parse(Unknown Source)
	at org.jabref@5.8.127/org.jabref.logic.xmp.XmpUtilShared.parseXmpMetadata(Unknown Source)
	at org.jabref@5.8.127/org.jabref.logic.xmp.XmpUtilReader.getXmpMetadata(Unknown Source)
	at org.jabref@5.8.127/org.jabref.logic.xmp.XmpUtilReader.readXmp(Unknown Source)
	at org.jabref@5.8.127/org.jabref.logic.importer.fileformat.PdfXmpImporter.importDatabase(Unknown Source)
	at org.jabref@5.8.127/org.jabref.logic.importer.fileformat.PdfMergeMetadataImporter.importDatabase(Unknown Source)
	at org.jabref@5.8.127/org.jabref.logic.importer.ImportFormatReader.lambda$importUnknownFormat$0(Unknown Source)
	at org.jabref@5.8.127/org.jabref.logic.importer.ImportFormatReader.importUnknownFormat(Unknown Source)
	at org.jabref@5.8.127/org.jabref.logic.importer.ImportFormatReader.importUnknownFormat(Unknown Source)
	at org.jabref@5.8.127/org.jabref.gui.importer.ImportAction.doImport(Unknown Source)
	at org.jabref@5.8.127/org.jabref.gui.importer.ImportAction.lambda$automatedImport$2(Unknown Source)
	at org.jabref@5.8.127/org.jabref.gui.util.BackgroundTask$1.call(Unknown Source)
	at org.jabref@5.8.127/org.jabref.gui.util.DefaultTaskExecutor$1.call(Unknown Source)
	at org.jabref.merged.module@5.8.127/javafx.concurrent.Task$TaskCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
```

</details>

**Comment by Siedlerchr** [comment]
Thanks for reporting this issue. I could reproduce this as well and will report the error to the XMPbox/pdfbox people as the erorr is coming from their library and I currently see no workaround from our side possible)

**Comment by Siedlerchr** [comment]
Corresponding bug report now created https://issues.apache.org/jira/browse/PDFBOX-5525

**Comment by THausherr** [comment]
Get a snapshot here:
https://repository.apache.org/content/groups/snapshots/org/apache/pdfbox/

**Comment by Siedlerchr** [comment]
@THausherr  Thanks for the quick fix. Is there any ETA for a new RC?

**Comment by Siedlerchr** [comment]
This issue was fixed with #10199 and is no longer reproducible
